### ISSUE TITLE
Old Nwchem site advertising questionable software? 

### ISSUE THREAD

**Issue by osbama** [nwchemgit/nwchem#275]
The first hit from google leads me to old nwchem site, which advertises some questionable software in between the text.

**Comment by jeffhammond** [comment]
Yes, it is a known issue.  Someone is squatting the old website.  We cannot force Google to update their index to find the new and correct site.  While it may be possible to file a DMCA takedown against the hijacked website, that's a nontrivial activity and nobody is signing up for it right now.
